build.boot

```diff
@@ -91,8 +91,10 @@ Type \"Yes\" to confirm."
   (let [env (System/getenv "WOMBATS_ENV")]
     (if (or (= env "")
             (nil? env))
-      "dev"
-      env)))
+      (do (System/setProperty "APP_ENV" "dev")
```

Comment:
What value does this give us? Isn't this the same as us using `(System/getenv "WOMBATS_ENV")` where the property is needed?